server/setLogLevel.xsl

```diff
@@ -17,6 +17,10 @@
         </xsl:copy>
     </xsl:template>
 
+    <xsl:template match="//log:handler[@name='FILE']" />
```

Comment:
Again, to remove the FILE logger which in a container is redundant as we should log to stdout/stderr.